src/Modal/Modal.js

```diff
@@ -98,11 +103,11 @@ export function Modal({ children, title, animationDuration, showClose, onClose,
   return (
     <ModalContext.Provider value={{ handleClose }}>
       <Portal>
-        <Transition in={isOpen} timeout={animationDuration}>
+        <Transition in={isOpen} timeout={animationDuration} onEntering={scrollToTop}>
           {state => (
             <FocusOn onEscapeKey={handleClose} enabled={isOpen}>
```

Comment:
@mathewmorris I think Escape is already supported here - was that not the case? Looking at the new escape support in the Header

Reply:
Oh that's for the close Icon if `Enter` is pressed while focused on the icon.